### ISSUE TITLE
Crowd simulation + Monte Carlo methods

### ISSUE THREAD

**Issue by sonjageorgievska** [NLeSC/TEAM2018#35]
Goal: In order to complete an existing manuscript, we need to perform validation of the method by simulation. Sonja + 1 or 2 persons with background in analytics/statistics would be enough.

Background: We have data about estimations of concert visitors'positions using Wi-Fi technology and their smart phones in (then) Amsterdam Arena. The data comes with a lot of errors and uncertainties. We have proposed a method for estimating the crowd density out of this data. The advantage of this method over related work is that the estimations becomes more accurate as the crowd size increases. This has been shown theoretically as a proof-of concept, but we do not have enough video data to confirm experimentally. Therefore, the idea is to validate the method with simulations. 

What needs to be done:

1. Use state of the art crowd simulator to simulate concert crowd with various increasing crowd densities.
2. Use the actual data of Wi-Fi location estimations to model the probability distributions of the errors in the estimations.
3. Draw samples from those distributions to introduce errors/uncertainties in the simulated data through time. 
4. Apply the proposed method to estimate the crowd density.
5. Compare the obtained crowd density with the method to the density as given by step 1. 
6. Check if the relative difference in step 5 becomes smaller as the crowd size increases. If it does (it should!), we have completed the paper :).

**Comment by romulogoncalves** [comment]
For the 4 days of the sprint the people to who this issue is assigned should report hours on:
crowd simulation project (@sonjageorgievska what is the project code)

For extra work you need to request time to the tech leads by sending an email to techleads@esciencecenter.nl

**Comment by romulogoncalves** [comment]
@sonjageorgievska could you let us know if the paper was submitted?
Thanks.

**Comment by romulogoncalves** [comment]
@sonjageorgievska would you like to use the November Sprint to finish the paper? Do you have hours for it?

@bldevries-escience would you be willing to help Sonja to finish it since you were there in the last Sprint?

**Comment by bldevries-escience** [comment]
> @sonjageorgievska would you like to use the November Sprint to finish the paper? Do you have hours for it?
> 
> @bldevries-escience would you be willing to help Sonja to finish it since you were there in the last Sprint?

Yes I would like to work on this project no problem. I already got into it last time. I probably have 2, maybe three days of the sprint available.

**Comment by romulogoncalves** [comment]
@bldevries-escience, Sonia will finish the paper until the sprint. During the sprint she plans to spend time on McFly. Hence, your expertise would be welcomed at SatSense for which you already showed your interest to join.

**Comment by romulogoncalves** [comment]
@sonjageorgievska were all the goals achieved? Could you summarize in 3 lines what was achieved? Can we consider it done and close this issue?

**Comment by sonjageorgievska** [comment]
Hi @romulogoncalves ,

Sorry, the github messages  were going to spam, I just created a new rule so I should be able to receive them.
The paper is ready for submission, we are contemplating on where to submit.
Therefore I will close the issue, thanks.
